<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>400</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>数据解密器 - v0.1.0</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>600</width>
     <height>104</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap>banner.png</pixmap>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>130</y>
     <width>60</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>PingFang SC</family>
     <kerning>true</kerning>
    </font>
   </property>
   <property name="text">
    <string>源文件</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="edit_line_src_file">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>150</y>
     <width>411</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>PingFang SC</family>
    </font>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_src_file">
   <property name="geometry">
    <rect>
     <x>460</x>
     <y>145</y>
     <width>113</width>
     <height>32</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>PingFang SC</family>
    </font>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="text">
    <string>选择文件</string>
   </property>
   <property name="checkable">
    <bool>false</bool>
   </property>
   <property name="autoDefault">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>200</y>
     <width>81</width>
     <height>16</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="font">
    <font>
     <family>PingFang SC</family>
    </font>
   </property>
   <property name="text">
    <string>解密后文件存放目录</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="edit_line_dest_file">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>220</y>
     <width>411</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>PingFang SC</family>
    </font>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_dest_file">
   <property name="geometry">
    <rect>
     <x>460</x>
     <y>215</y>
     <width>113</width>
     <height>32</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>PingFang SC</family>
    </font>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="text">
    <string>选择目录</string>
   </property>
   <property name="autoDefault">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>270</y>
     <width>71</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>PingFang SC</family>
    </font>
   </property>
   <property name="text">
    <string>密钥</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="edit_line_key">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>290</y>
     <width>101</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>PingFang SC</family>
    </font>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_decrypt">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>340</y>
     <width>113</width>
     <height>32</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>PingFang SC</family>
    </font>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="text">
    <string>开始解密</string>
   </property>
   <property name="autoDefault">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QLabel" name="lbl_opendir">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>200</y>
     <width>81</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>PingFang SC</family>
     <pointsize>13</pointsize>
     <underline>true</underline>
     <strikeout>false</strikeout>
    </font>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="text">
    <string>打开目录</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
